aztk_sdk/client.py

```diff
@@ -0,0 +1,160 @@
+from datetime import datetime, timedelta
+import azure.batch.models as batch_models
+import aztk_sdk.utils.azure_api as azure_api
+import aztk_sdk.utils.helpers as helpers
+import aztk_sdk.utils.constants as constants
+import aztk_sdk.utils.get_ssh_key as get_ssh_key
+import aztk_sdk.models as models
+
+
+class Client:
+    def __init__(self, secrets_config: models.SecretsConfiguration):
+        self.secrets_config = secrets_config
+
+        self.blob_config = azure_api.BlobConfig(
+            account_key=self.secrets_config.storage_account_key,
+            account_name=self.secrets_config.storage_account_name,
+            account_suffix=self.secrets_config.storage_account_suffix
+        )
+        self.batch_config = azure_api.BatchConfig(
+            account_key=self.secrets_config.batch_account_key,
+            account_name=self.secrets_config.batch_account_name,
+            account_url=self.secrets_config.batch_service_url
+        )
+
+        self.batch_client = azure_api.make_batch_client(self.batch_config)
+        self.blob_client = azure_api.make_blob_client(self.blob_config)
+    
+    def __delete_pool_and_job(self, cluster_id: str):
```

Comment:
You have comments for some functions but not others. Please add them everywhere.